packages/fhir-care-team/src/components/CreateEditCareTeam/index.tsx

```diff
@@ -120,9 +120,12 @@ const CreateEditCareTeam: React.FC<CreateEditCareTeamProps> = (props: CreateEdit
       })) ?? [],
   };
 
+  if (fhirPractitioners.isError || singleCareTeam.isError || fhirGroups.isError)
+    return <BrokenPage />;
+
   if (
-    fhirPractitioners.isLoading ||
-    fhirGroups.isLoading ||
+    !fhirPractitioners.isSuccess ||
+    !fhirGroups.isSuccess ||
```

Comment:
@ashfahan why?

Reply:
@ashfahan  The problem I see here is that the page will go to broken in cases where we have an initial successful render, but there is an error during refetch, see https://github.com/tannerlinsley/react-query/pull/1108#issuecomment-718474623